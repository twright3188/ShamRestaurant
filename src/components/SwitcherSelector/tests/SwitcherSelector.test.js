import React from 'react';
import renderer from 'react-test-renderer';
import { SwitcherSelector } from '../index';

it('switcher render', () => {
    const tree = renderer.create(<SwitcherSelector options={[{ label: '1', value: 0 }]} thirdPerson/>).toJSON();
    expect(tree).toMatchSnapshot();
});