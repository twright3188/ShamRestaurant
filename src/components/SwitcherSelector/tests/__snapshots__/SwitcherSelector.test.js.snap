// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`switcher render 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "row",
      },
      Object {
        "borderColor": "#399AE5",
        "borderRadius": 5,
        "borderWidth": 1,
        "opacity": 0.7,
      },
    ]
  }
>
  <View
    onMoveShouldSetResponder={[Function]}
    onMoveShouldSetResponderCapture={[Function]}
    onResponderEnd={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderReject={[Function]}
    onResponderRelease={[Function]}
    onResponderStart={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onStartShouldSetResponderCapture={[Function]}
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      onLayout={[Function]}
      style={
        Object {
          "backgroundColor": "#FFFFFF",
          "borderRadius": 5,
          "height": 32,
        }
      }
    >
      <View
        style={
          Object {
            "borderColor": "#399AE5",
            "borderRadius": 5,
            "borderWidth": 0,
            "flex": 1,
            "flexDirection": "row",
          }
        }
      >
        <View
          accessible={true}
          clickable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "center",
              "opacity": 1,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "backgroundColor": "transparent",
                  "color": "#399AE5",
                  "fontSize": 14,
                  "fontWeight": "normal",
                  "textAlign": "center",
                },
                Object {},
              ]
            }
          >
            1
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
